main10.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import pandas as pd
+import json
+
+URL = "https://raw.githubusercontent.com/josago97/EurovisionDataset/refs/heads/main/dataset/eurovision.json"
```

Comment:
Betra væri að vísa í official release útgáfu, t.d. frá https://github.com/josago97/EurovisionDataset/releases/tag/2024.1
```suggestion
URL = "https://github.com/josago97/EurovisionDataset/releases/download/2024.1/eurovision.json"
```